internal/decryption/kmip/client.go

```diff
@@ -0,0 +1,386 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kmip
+
+import (
+	"bufio"
+	"fmt"
+
+	"crypto/tls"
+	"crypto/x509"
+
+	kmip "github.com/gemalto/kmip-go"
+	"github.com/gemalto/kmip-go/kmip14"
+	"github.com/gemalto/kmip-go/kmip20"
+	"github.com/gemalto/kmip-go/ttlv"
+	"github.com/pkg/errors"
```

Comment:
remove the new lines here and use go imports

Reply:
done, thx